### PR TITLE
Joe/new methods

### LINKED ISSUE #67: Add missing evm-tableland methods
Related to #56. Add support for other missing methods:

- `setController`
- `getController`
- `lockController`

### PR DESCRIPTION
**Pull request by joewagner**
Fixes #67 

This adds `getController`, `setController`, and `lockController`.  The Validator RPC does not yet support these fully, but direct to contract is working.

### CODE REVIEW

File: src/lib/connection.ts
Comment (carsonfarmer) [inline src/lib/connection.ts:97]: Just wondering out loud... should we namespace all of these under controller? so controller.get, controller.set, controller.lock?

File: src/lib/eth-calls.ts
Comment (carsonfarmer) [inline src/lib/eth-calls.ts:93]: Why diff from above. Just because this is an actual mutating call?

File: src/lib/get-controller.ts
Comment (carsonfarmer) [inline src/lib/get-controller.ts:2]: Causing a lint error. Maybe just remove for now?

File: src/lib/get-controller.ts
Comment (carsonfarmer) [inline src/lib/get-controller.ts:18]: Linter doesn't like this.

File: src/lib/tableland-calls.ts
Comment (carsonfarmer) [inline src/lib/tableland-calls.ts:42]: Needs a linter escape

**Review by carsonfarmer — COMMENTED** [review]
Looking good so far @joewagner. Obvi some linter issues, and we'll need to decide if the RPC api needs to support these or not?

File: src/lib/eth-calls.ts
Comment (joewagner) [reply src/lib/eth-calls.ts:93]: Yeah, since getController is a `view` it returns a string. setController and lockController return transactions, and we've been returning the hash in those cases.

File: src/lib/connection.ts
Comment (joewagner) [reply src/lib/connection.ts:97]: That would work.  My tendency is usually to try to keep the api intuitive to a developer when possible. Currently the Contract functions, RPC endpoints, and sdk methods all match.  This makes sense to me, but maybe a `controller` object with functions on it would be better?
If we do switch, i think we would want to make sure to bind the context of the functions to "tableland", e.g. inside of tableland.controller.set we would want `this` to be tableland not controller so we can call `this.checkNetwork` etc...  Alternatively we could abstract the idea of a `super` as `this.super` (or something like that) that points to tableland, but that seems more complicated.

**Comment by joewagner** [comment]
@carsonfarmer sorry about all the linting issues.  Was kinda stuck until we decided if we want to have the RCP server relay all these methods.  
We have been discussing having the RPC server relay creating tables.  If we are doing that, it seems like these methods probably make sense too.  This winter when we discussed relay vs. direct it seemed like we wanted to mirror what the network might do once we work out all the validator incentives etc...  Should we do a TPD for what gets relayed and what doesn't?

**Review by carsonfarmer — APPROVED** [review]
Sorry for the delay here @joewagner things have been hectic. Don't wanna hold this one up any longer though! Looks good to me.

**Comment by joewagner** [comment]
> Sorry for the delay here @joewagner things have been hectic. Don't wanna hold this one up any longer though! Looks good to me.

No problem!  Definitely a lot of more important things happening recently...
I'll merge this now, do we have plans for when the next release will be?  I think some of the pilot program folks could use some of the features that we've built out recently.

**Pull request merged**
